docs/static/spec/openapi/logstash-api.yaml

```diff
@@ -32,52 +32,195 @@ security:
 tags:
   - name: health
     x-displayName: Health report
-    # description: 
+    # description:
     # externalDocs:
-    #   description: 
-    #   url: 
+    #   description:
+    #   url:
   - name: hot threads
     x-displayName: Hot threads
-    # description: 
+    # description:
     # externalDocs:
-    #   description: 
-    #   url: 
+    #   description:
+    #   url:
   - name: node info
     x-displayName: Node info
     description: |
       Logstash node info, like pipeline settings, OS info, and JVM info.
     # externalDocs:
-    #   description: 
-    #   url: 
+    #   description:
+    #   url:
   - name: node stats
     x-displayName: Node stats
     description: |
       Node stats, like JVM stats, process stats, event-related stats, and pipeline runtime stats.
     # externalDocs:
-    #   description: 
-    #   url: 
+    #   description:
+    #   url:
   - name: plugin info
     x-displayName: Plugins info
     description: |
       Plugin info, including a list of installed plugins.
     # externalDocs:
-    #   description: 
-    #   url: 
-paths: 
-  /_node/<types>:
+    #   description:
+    #   url:
+paths:
+  /_node/jvm:
     get:
-      summary: Get node info
+      summary: Gets node-level JVM info, including info about threads.
       description: |
-        Get information about Logstash nodes, where `<types>` (optional) specifies the types of node info you want returned.
-        You can limit the info that is returned by combining any of these types in a comma-separated list:
+        Show node-level JVM stats, such as the JVM process id, version, VM info, memory usage, and info about garbage collectors
+      operationId: nodeInfoJVM
+      tags:
+        - node info
+      parameters:
+        - name: pretty
+          in: query
+          schema:
+            type: boolean
+          description: >
+            If you append `?pretty=true` to the request, the JSON returned will be pretty formatted. Use it for debugging only!
+      responses:
+        '200':
+          description: Indicates a successful call
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/Common'
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - type: object
+                    properties:
+                      jvm:
+                        type: object
+                        properties:
```

Comment:
>                        properties:

Ideally each of these properties (here and in other objects) will also have a description, but it's okay to add these in subsequent PRs as necessary.

Reply:
Some properties are worth a description if the field name is not self descriptive